delfin/task_manager/scheduler/schedulers/telemetry/job_handler.py

```diff
@@ -119,7 +121,9 @@ def schedule_job(self, task_id):
                 history_on_reschedule = CONF.telemetry. \
                     performance_history_on_reschedule
                 # Adjust start_time and end_time based on last_run_time
-                end_time = current_time * 1000
+                timestamp_offset = self.driver_api.get_timestamp_offset(
```

Comment:
catch not implemented exception here and default to 0 if driver has not implemented  get_timestamp_offset

Reply:
Done.
1. catch NotImplementedError exception.
2. default to 0 if NotImplementedError catched.